internal/pkg/cli/task_run.go

```diff
@@ -199,10 +200,11 @@ func (o *runTaskOpts) configureRunner() (taskRunner, error) {
 		}, nil
 	}
 
-	return &task.NetworkConfigRunner{
+	return &task.ConfigRunner{
 		Count:     o.count,
 		GroupName: o.groupName,
 
+		Cluster:        "data-api-expr-tutorial",
```

Comment:
I'm not familiar with this area; just wondering why this is getting hard-coded.

Reply:
ah oh it was done by accident.. i just wanted to make sure that the value went through so I hard-coded an existing cluster to see if it works. updated!

Reply:
😄